_data/toc.yaml

```diff
@@ -1561,66 +1561,78 @@ manuals:
   section:
     - path: /build/
       title: Overview
+    - path: /build/install-buildx/
+      title: Install Buildx
     - sectiontitle: Building images
       section:
         - path: /build/building/packaging/
           title: Packaging your software
         - path: /build/building/context/
           title: Build context
-        - sectiontitle: Build drivers
-          section:
-            - path: /build/building/drivers/
-              title: Overview
-            - path: /build/building/drivers/docker/
-              title: Docker driver
-            - path: /build/building/drivers/docker-container/
-              title: Docker container driver
-            - path: /build/building/drivers/kubernetes/
-              title: Kubernetes driver
-            - path: /build/building/drivers/remote/
-              title: Remote driver
-        - sectiontitle: Cache
-          section:
-            - path: /build/building/cache/
-              title: Optimizing builds with cache
-            - path: /build/building/cache/garbage-collection/
-              title: Garbage collection
-            - sectiontitle: Cache backends
-              section:
-                - path: /build/building/cache/backends/
-                  title: Overview
-                - path: /build/building/cache/backends/inline/
-                  title: Inline
-                - path: /build/building/cache/backends/local/
-                  title: Local
-                - path: /build/building/cache/backends/registry/
-                  title: Registry
-                - path: /build/building/cache/backends/gha/
-                  title: GitHub Actions
-                - path: /build/building/cache/backends/azblob/
-                  title: Azure Blob Storage
-                - path: /build/building/cache/backends/s3/
-                  title: Amazon S3
-        - sectiontitle: Exporters
-          section:
-          - path: /build/building/exporters/
-            title: Overview
-          - path: /build/building/exporters/image-registry/
-            title: Image and registry exporters
-          - path: /build/building/exporters/local-tar/
-            title: Local and tar exporters
-          - path: /build/building/exporters/oci-docker/
-            title: OCI and Docker exporters
         - path: /build/building/multi-stage/
           title: Multi-stage builds
         - path: /build/building/multi-platform/
           title: Multi-platform images
         - path: /build/building/base-images/
           title: Create your own base image
-    - sectiontitle: Customizing builds
+        - path: /build/building/multiple-builders/
+          title: Using multiple builders
```

Comment:
I think we should get rid of this page. WDYT @jedevc?

Reply:
Yup, I don't think there's much content here worth preserving - maybe we should merge it into the "Drivers Overview" page, and then delete this one?